src/CommandParser.h

```diff
@@ -7,10 +7,6 @@
 //
 // Screaming Udder!                              https://esss.se
 
-/// \file This file defines the different success and failure status that the
-/// `StreamMaster` and the `Streamer` can incur. These error object have some
-/// utility methods that can be used to test the more common situations.
-
```

Comment:
This comment appears to be copy-pasted from StreamStatus and doesn't belong here at all, so I deleted it.